README.md

```diff
@@ -1,8 +1,16 @@
-# SlimFast [![Build Status](https://travis-ci.org/doomspork/slim_fast.png?branch=master)](https://travis-ci.org/doomspork/slim_fast) [![Hex Version](https://img.shields.io/hexpm/v/slim_fast.svg)](https://hex.pm/packages/slim_fast) [![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+# SlimFast [![Build Status][travis-img]][travis] [![Hex Version][hex-img]][hex] [![License][license-img]][license]
+
+[travis-img]: https://travis-ci.org/doomspork/slim_fast.png?branch=master
+[travis]: https://travis-ci.org/doomspork/slim_fast
+[hex-img]: https://img.shields.io/hexpm/v/slim_fast.svg
+[hex]: https://hex.pm/packages/slim_fast
+[license-img]: http://img.shields.io/badge/license-MIT-brightgreen.svg
+[license]: http://opensource.org/licenses/MIT
 
 > A refreshing way to slim down your markup in Elixir.
 
-SlimFast is an [Elixir](http://elixir-lang.com) library for rendering [Slim](http://slim-lang.com) templates as HTML.
+SlimFast is an [Elixir](http://elixir-lang.com) library for rendering
+[Slim](http://slim-lang.com)-like templates as HTML.
```

Comment:
Let's remove the Phoenix section now that it's belong the reference and update this to something like:  

"SlimFast is an [Elixir](elixir) library for rendering [Slim](slim)-like templates as HTML.  For use with [Phoenix][phoenix], please see [PhoenixSlim][phoenix-slim]."

[elixir]: http://elixir-lang.com
[phoenix-slim]: https://github.com/doomspork/phoenix_slim
[phoenix]: http://www.phoenixframework.org
[slim]: http://slim-lang.com

Thoughts?

Thanks again @lpil :+1: